test/can-dom-mutate-test.js

```diff
@@ -7,7 +7,7 @@ var test = unit.test;
 var moduleMutationObserver = testUtils.moduleMutationObserver;
 
 moduleMutationObserver('can-dom-mutate', function () {
-	test('onNodeInsertion should be called when that node is inserted', function (assert) {
+	test('onNodeInsertion  ', function (assert) {
```

Comment:
Undo this change, and everything should be good. 👍

Reply:
oops! thanks for catching.